bin/hdfcoinc/pycbc_multiifo_page_foreground

```diff
@@ -0,0 +1,161 @@
+tc = [foreground[ifo + '/time'][:][ii_sort] for ifo in detectors]
+n_ifos = [np.count_nonzero(np.array(ct) > 0) for ct in zip(*tc)]
+mean_time = [np.mean(np.array(ct)[np.array(ct) > 0]) for ct in zip(*tc)]
+
+# Add GPS event time and n_ifos to the table
+cols.append(mean_time)
+names.append("End Time (GPS)")
+formats.append("#.##")
+
+cols.append(n_ifos)
+names.append("No. IFOs in coinc")
+formats.append("#")
+
+# Get template information - mass, spin etc
+tid = foreground['template_id'][:][ii_sort]
+with h5py.File(args.bank_file, 'r') as bank_f:
+    mass1 = bank_f['mass1'][:][tid]
+    mass2 = bank_f['mass2'][:][tid]
+    spin1z = bank_f['spin1z'][:][tid]
+    spin2z = bank_f['spin2z'][:][tid]
+mchirp, eta = pycbc.pnutils.mass1_mass2_to_mchirp_eta(mass1, mass2)
+cols += [mchirp, mass1, mass2, spin1z, spin2z]
+names += ['mchirp', 'm1', 'm2', 's1z', 's2z']
+formats += ['##.##', '##.##', '##.##', '##.##', '##.##']
+
+# Get time difference information
+det_two_combo= np.array(list(combinations(detectors,2)))
+tdiff = []
+tdiff_str = []
+tdiff_format =[]
+for i in range(len(det_two_combo)):
+     time_1 = np.array(foreground[det_two_combo[i,0]+'/time'][:][ii_sort])
+     time_2 = np.array(foreground[det_two_combo[i,1]+'/time'][:][ii_sort])
+     tdiff_vals = (time_1 - time_2) * 1000
+     tdiff_vals[np.logical_or(time_1 < 0, time_2 < 0)] = np.nan
+     tdiff_1 = ['%.2f' % td if not np.isnan(td) else ' ' for td in tdiff_vals]
+     tdiff.append(tdiff_1)
+     tdiff_head= '%s - %s time (ms)' % (det_two_combo[i,0], det_two_combo[i,1])
+     tdiff_str.append(tdiff_head)
+     tdiff_format.append('##.##')
+
+cols += tdiff
+names += tdiff_str
+formats += tdiff_format
+
+
+ids = {detector:foreground[detector+'/trigger_id'][:][ii_sort] for detector
+       in detectors}
+f.close()
+
+stfs = {detector: [h5py.File(stf_name, 'r') for stf_name in
+                   args.single_detector_triggers
+                   if detector in h5py.File(stf_name, 'r')][0]
+       for detector in detectors}
+
+for ifo in detectors:
+    ids_ifo = np.array(ids[ifo])
+    stf_ifo = stfs[ifo][ifo]
+    snr_vals = stf_ifo['snr'][:][ids_ifo]
+    snr_vals[ids_ifo == -1] = np.nan
+    chisq_dof = stf_ifo['chisq_dof'][:][ids_ifo]
+    chisq_vals = stf_ifo['chisq'][:][ids_ifo] / (2 * chisq_dof - 2)
+    chisq_vals[ids_ifo == -1] = np.inf
+    newsnr_vals = pycbc.events.ranking.newsnr(snr_vals, chisq_vals)
+    chisq_vals[ids_ifo == -1] = np.nan
+    newsnr_vals[ids_ifo == -1] = np.nan
+    snr = ['%.2f' % s if not np.isnan(s) else ' ' for s in snr_vals]
+    chisq = ['%.2f' % c if not np.isnan(c) else ' ' for c in chisq_vals]
+    newsnr = ['%.2f' % s if not np.isnan(s) else ' ' for s in newsnr_vals]
+
+    names += [ifo + " SNR", ifo + " Red. Chisq", ifo + " NewSNR"]
+    cols += [snr, chisq, newsnr]
+    formats += ['##.##', '##.##', '##.##']
+
+columns = [np.array(col) for col in cols]
+html_table = pycbc.results.html_table(columns, names,
```

Comment:
I will just add a line before it to just stick to the convention: 
``` logging.info('Making table of foreground triggers')```
[Link](https://github.com/gwastro/pycbc/blob/bbbec62476481ec1885b1068a9a956832645bd5e/bin/hdfcoinc/pycbc_page_foreground#L159)

Reply:
I removed the logging module as it was completely unused
I will add it in as a print statement though

Reply:
I also prefer print. I never liked logging info 🥇

Reply:
You'll hate the latest commit then - I've added a lot of logging.

logging.info is better for workflows as otherwise you'll print a lot of debug information into .err files, the disk space usage will add up

Reply:
I actually highly recommend the `logging` module for progress/debug/info over `print()`. My favorite argument is that it gives timestamps for free, which can be *very* useful for debugging, but there are [other arguments](https://stackoverflow.com/questions/6918493/in-python-why-use-logging-instead-of-print).